tests/modules/audit_logs/resources/test_audit_log.py

```diff
@@ -173,7 +173,7 @@ def test_most_ia_pipeline_audit_log(
 
     assert expected_sighting in sighting_audit_items.json
     assert expected_encounter in encounter_audit_items.json
-    assert expected_annotation in annotation_audit_items.json
+    assert expected_annotation not in annotation_audit_items.json
```

Comment:
Yes, if we can no longer know what the annotation guid, this line plus the two things it's checking are not needed anymore.

Reply:
ok, i will go ahead and just remove the annot audit_log test then.  thanks.